Bylaws.md

```diff
@@ -181,8 +200,7 @@ Interested Parties will not be required to pay anything for their participation
 
 The Forum shall maintain a Member Mail List and Member Web Site that are not accessible by the public. The following matters may be posted to the Member Mail List and Member Web Site:
 
- 1. Draft minutes of Forum meetings (both virtual and in-person, and including any sub-groups or committees) will be posted to the Member Mail List to allow Members and Associate Members to make sure they are being correctly reported.
-    Minutes will be considered final when approved at a subsequent Forum Meeting or Forum Teleconference; provided, however, that if there is no Forum Meeting or Forum Teleconference scheduled within three (3) weeks of the publication of the draft minutes. Final minutes will then be posted to the Public Mail List and Public Web Site. The Chair will, upon request, make redactions of any part of the public copy of the minutes identified as private or sensitive by either the information discloser or a member mentioned or affiliated with the subject of the information.
+ 1. Draft minutes of Forum Meetings and Forum Teleconferences will be posted to the corresponding Member Mail List to allow Members to make sure they are being correctly reported. Sub-groups and committees may also post to a corresponding Member Mail List. Minutes will be considered final when approved at a subsequent Forum Meeting or Forum Teleconference; provided, however, that if there is no Forum Meeting or Forum Teleconference scheduled within three (3) weeks of the publication of the draft minutes. Final minutes will then be posted to the corresponding Public Mail List or if it relates to a sub-group or committee, they will be posted to the corresponding Public Mail List of the sub-group or committee. Final minutes of Forum meetings or Forum Teleconferences will additionally be posted to the Public Web Site. The Chair will, upon request, make redactions of any part of the public copy of the minutes identified as private or sensitive by either the information discloser or a member mentioned or affiliated with the subject of the information.
```

Comment:
`Minutes will be considered final when approved at a subsequent Forum Meeting or Forum Teleconference; provided, however, that if there is no Forum Meeting or Forum Teleconference scheduled within three (3) weeks of the publication of the draft minutes.` ->
`Minutes will be considered final when approved at a subsequent Forum Meeting or Forum Teleconference, except if there is no Forum Meeting or Forum Teleconference scheduled within three (3) weeks of the publication of the draft minutes, in which case Minutes will be considered final three (3) weeks after publication of the draft minutes.`

I'm not sure that's what was originally intended, so replace `three (3) weeks after publication of the draft minutes.` with whatever the correct criteria are.

Reply:
Addressed in 090c6289f42e46e535db392274bb8fe251ee05fd